### PR TITLE
Asintel 720 import app redundancies to dm

### PR DESCRIPTION
**Pull request by uhimabindu**

### CODE REVIEW

**Review by sarmaevanidfe — COMMENTED** [review]
looks fine to me

File: pipeline/Master-ImportAccUsersComtFinResvToDataMart.json [hunk 1]
```diff
@@ -246,7 +246,7 @@
 				"type": "SqlServerStoredProcedure",
 				"dependsOn": [
 					{
-						"activity": "ExecuteImportReservationToDM",
+						"activity": "ExecuteImportRedundancyToDataMart",
```
Comment (arvinthseran) [inline pipeline/Master-ImportAccUsersComtFinResvToDataMart.json:249]: can you review the activity name please

File: pipeline/Master-ImportAccUsersComtFinResvToDataMart.json [hunk 2]
```diff
@@ -276,6 +276,31 @@
 					"referenceName": "datamgmt_staging_dbconnectionstring",
 					"type": "LinkedServiceReference"
 				}
+			},
+			{
+				"name": "ExecuteImportRedundancyToDataMart",
+				"type": "ExecutePipeline",
+				"dependsOn": [
+					{
+						"activity": "ExecuteImportReservationToDM",
```
Comment (arvinthseran) [inline pipeline/Master-ImportAccUsersComtFinResvToDataMart.json:285]: can you review the activity name please

**Review by arvinthseran — APPROVED** [review]

**Review by arvinthseran — APPROVED** [review]

**Review by sarmaevanidfe — COMMENTED** [review]
look fine to me

**Review by sarmaevanidfe — COMMENTED** [review]
Changes looks fine to me

**Review by sarmaevanidfe — COMMENTED** [review]
looks fine to me

**Review by sarmaevanidfe — APPROVED** [review]

**Pull request merged**